apps/govuk-docs/check-links.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash +x
+
+set -e
```

Comment:
`-euo pipefail`?

Reply:
Oh maybe not, you disable this later.

Reply:
Yes, there's more than one thing running here, so the exit code needs to be set after clean-up, and reflect the outcome of the test process.